static/styles/rendered_markdown.scss

```diff
@@ -354,6 +354,56 @@
         padding: 5px;
         border: none;
     }
+
+    .admonition {
+        .admonition-title + * {
+            margin-top: 10px;
+        }
+
+        p.admonition-title {
+            font-weight: bold;
+        }
+
+        &.admonition-danger,
+        &.admonition-error,
+        &.admonition-critical {
```

Comment:
I think it's OK to have some synonyms in the syntax, but it'd be best to canonicalize them to a single name in the rendered HTML. That way the interface for clients to implement (e.g. what they have to cover with CSS rules) is simpler. In particular if we add different synonyms later, that approach would let them work correctly with existing clients.